src/sass/_theme_rst.sass

```diff
@@ -220,6 +220,9 @@
     background: $highlight-color
     display: inline-block
     font-weight: bold
+    font-style: italic
```

Comment:
I'm -1 on making search results italic. They do jump out more, but it change the formatting in a way not intended by the author. IMHO we should stick to colors only.

Reply:
I'm neutral on using italics, but italics + underline + background color change does feel like we've thrown a lot at the highlight style. I'd be fine starting with just the color and underline change and skipping italics.

Reply:
I made this change.